nixos/modules/services/web-apps/outline.nix

```diff
@@ -0,0 +1,784 @@
+{ config, lib, pkgs, ...}:
+
+let
+  inherit (builtins) concatStringsSep toString;
+  inherit (lib)
+    escapeShellArg
+    literalExpression
+    mkEnableOption
+    mkIf
+    mkMerge
+    mkOption
+    optional
+    optionalAttrs
+    optionalString
+    types;
```

Comment:
Maintaining such lists is a bit pointless. Prefixing everything with lib/builtins or using with is easier to maintain.